R/cleanup.R

```diff
@@ -713,37 +701,37 @@ cleanGeneDescription <- function(prot, column) {
 #' selectLongestDuplicate()
 #' }
 selectLongestDuplicate <- function(prot, column) {
-    col <- sym(column)
-
-    prot$row.orig <- 1:nrow(prot)
-
+    col <- rlang::sym(column)
+    prot <- prot %>% 
```

Comment:
Code Structure & Functionality
Parameter Handling:

The column parameter is converted to a symbol using rlang::sym, enabling flexible selection of any column by name. This is useful for adaptability across datasets with varying column names.